index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/responsive.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div id="container">
+      <header>
+        <nav>
+          <p id="logo">Lisandro Seia</p>
+
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <section id="present">
+          <div id="titles">
+            <h1>
+              I’m Lisandro <br />
+              Glad to see you!
+            </h1>
+            <p id="present-text">
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div id="contact">
+            <p id="contact-p">LET’S CONNECT</p>
+            <div id="icons">
+              <a target="blank" href="https://twitter.com/LisandroSeia"
+                ><img alt="twitter" src="/assets/imgs/twitter.png"
+              /></a>
+              <a
+                target="blank"
+                href="https://www.linkedin.com/in/lisandro-seia-295120225/"
+                ><img alt="linkedin" src="/assets/imgs/linkedin.png"
+              /></a>
+              <a href="#"><img alt="medium" src="/assets/imgs/medium.png" /></a>
+              <a target="blank" href="https://github.com/lisandroseia"
+                ><img alt="github" src="/assets/imgs/github.png"
+              /></a>
+              <a href="#"
+                ><img alt="angellist" src="/assets/imgs/angellist.png"
+              /></a>
```

Comment:
- [ ] Hi, you've done well to include the icons from the Figma. Please make sure that the links you use are working correctly. it seems that you images don't work correctly. please 🙏 fix this.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143321244-19001a34-e2d7-4224-934b-d39342b8d188.png)
## Figma Design
![image](https://user-images.githubusercontent.com/18730106/143321381-a5a1aec7-2f10-411f-87f0-9db9ec5f48d0.png)

i think you just need to change the link you referenced to begin with a dot (`.`)
something like this `<img alt="angellist" src="./assets/imgs/angellist.png"/>`